docs/quality-assurance/test-repositories.md

```diff
@@ -0,0 +1,15 @@
+# Test GitHub Repositories
+
+This page lists the UKHO test-related repositories on GitHub.
+
+## Demo API Test Framework
+
+Example Java project for API testing, using REST-assured and obtaining secrets from an Azure Key Vault.
+
+https://github.com/UKHO/demo-api-test-framework
```

Comment:
Links should be inside <> in markdown rather than relying upon GitHubs automatic link parser - this makes this project portable to any reader